src/component/TitleBar.tsx

```
@@ -30,34 +29,53 @@ export default function TitleBar() {
                 </div>
-                {/* <div onClick={maxHandler}>
-                    max
-                </div> */}
-                <div onClick={closeHandler} className="hover:bg-accent-negative h-full flex items-center px-2 transition-colors">
-                    <img src={closeButton} className="pointer-events-none text-text-primary" alt="close" />
+                {import.meta.env.DEV && <div onClick={maxHandler}>max</div>}
+                <div
+                    onClick={closeHandler}
+                    onKeyDown={(e) => e.key === "Enter" && closeHandler()}
+                    tabIndex={0}
+                    role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)